build

```diff
@@ -278,6 +278,18 @@ def list_targets(config):
     for target in sorted(config['vagrant-targets']):
         message('vagrant\t%s' % target)
 
+def get_host_platform(config):
+    arch = platform.machine()
+    return config['host-platforms'][arch]
+
+def get_qemu_platform(host_platform, target_platform):
+    # workaround https://bugs.launchpad.net/qemu/+bug/1805913 for 32-bit targets
+    if host_platform == 'linux/amd64':
+        return 'linux/amd64' if target_platform not in ('linux/arm/v5', 'linux/arm/v7') else 'linux/386'
+    if host_platform == 'linux/arm64/v8':
```

Comment:
I assume that ARM64 requires the same special treatment as AMD64 when building for a 32-bit target.